src/main/resources/edu/cornell/kfs/module/ld/businessobject/datadictionary/PositionData.xml

```diff
@@ -57,45 +57,46 @@
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="departmentId" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.orgCode" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
-  </bean>
-  
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:insertAfter-ref="PositionData-extension-inactivationDate"/>
+
+          <!-- Bean override for legacy modal style presentation -->
+          <bean parent="FieldOverrideForListElementInsert">
+            <property name="propertyName" value="inquirySections[0].inquiryFields" />
+            <property name="propertyNameForElementCompare" value="attributeName" />
+            <property name="element">
+              <bean parent="FieldDefinition" p:attributeName="departmentId"/>
+            </property>
+            <property name="insertAfter">
+              <list>
+                <bean parent="FieldDefinition" p:attributeName="extension.orgCode"/>
+              </list>
+            </property>
+          </bean>
+          <bean parent="FieldOverrideForListElementInsert">
+            <property name="propertyName" value="inquirySections[0].inquiryFields" />
+            <property name="propertyNameForElementCompare" value="attributeName" />
+            <property name="insertAfter">
+              <list>
+                <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate"/>
+              </list>
+            </property>
+          </bean>
+      </list>
+    </property>
   </bean>
 
 <!-- Business Object Lookup Definition -->
```

Comment:
One of my earlier comments mentioned a potential simplification to the **lookup** bean overrides as well, but if the PositionData lookup overrides still work as expected, then I'm okay with deferring the lookup cleanup to the KFSPTS-26093 user story.